### PR TITLE
feat: rendering-metadata-in-paymentsummary

### PR DESCRIPTION
**Pull request by pavanjoshi914**
rendering metadata in payment Summary

image

https://user-images.githubusercontent.com/55848322/177002589-a6901746-8ad0-4370-87e3-5036eb8b1aab.mp4

### CODE REVIEW

**Comment by bumi** [comment]
@pavanjoshi914 can you solve this conflict?

File: src/app/components/PaymentSummary.tsx [hunk 1]
```diff
@@ -6,6 +6,30 @@ type Props = {
...
+              ) : (
+                metadata[key]
+              )}
+            </dd>
+          </>
+        );
+      }
+    }
+  } else {
+    metadataElements.push(<p>&quot;No Metadata Present&quot;</p>);
```
Comment (bumi) [inline src/app/components/PaymentSummary.tsx:30]: I think at some point we can leave this and just show nothing. 
this does not really give much value to the user.

Comment (pavanjoshi914) [reply src/app/components/PaymentSummary.tsx:30]: sure!

Comment (pavanjoshi914) [reply src/app/components/PaymentSummary.tsx:30]: Resolved in this pull request::  https://github.com/getAlby/lightning-browser-extension/pull/1112

**Pull request merged**